### DISCUSSION TITLE
single or multi pick tag

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#44] [Ideas]
Last time I made this obervsations working with tags:
In the Grasshopper version 2033: If cmd "Add Tag" was executed you could single click a stored tag and it was autosubmitted.
Since the Grasshopper version 2035: If cmd "Add Tag" is executed it does not autosubmit anymore when a tag is clicked. Instead you can multi select more than one tag. This is indeed handsome if you need more than one tag but it increases clicks if only one tag is needed.

I would really like to choose between these two possibilites above: Either a new option in the settings page (version 2033 behavior or 2035 and after behavior) or implement a new behavior: Left Click: single select tag and autosubmit. Right Click: multi select tag and don't autosubmit.

**Comment by madprops** [comment]
I think i'll add a setting, since the behavior you seek is specific.

Or maybe i'll make it the default and only behavior....

**Reply by N3C2L** [reply]
Thanks. I have a light RSI syndrome and need to reduce clicks everywhere. I need many breaks because my fingers hurt sometime.

**Reply by madprops** [reply]
I see. It should be one single click, on next version.

**Comment by N3C2L** [comment]
Excellent, thanks.
